packages/styles/stories/Components/dialog/dialog.stories.js

```diff
@@ -195,4 +196,14 @@ VerticalFormInDialog.parameters = {
       story: 'Verical Form displays all the components aligned vertically. It is created by positioning the `fd-form-label` class above the `fd-input` class in seperate rows.'
     }
   }
+};
+
+export const DeviceSpecifications = () => deviceSpecificationsExampleHtml;
+DeviceSpecifications.storyName = 'Tablet and Hybrid Device Specification';
```

Comment:
Can you name the story const same as story name and remove `DeviceSpecifications.storyName` assignment?